0800-appendix-197653-perow.Rmd

```diff
@@ -0,0 +1,288 @@
+my_site <- 197653
+my_site2 <- 197657
+
+```
+
+`r fpr::fpr_appendix_title()`
+
+## Site Location {.unnumbered}
+
+PSCIS crossing `r as.character(my_site)` is located on
+`r fpr::fpr_my_pscis_info()` approximately 10km west of Topley, BC.
+PSCIS crossing `r as.character(my_site)` is located on
+`r fpr_my_pscis_info(col_pull = road_name)`, approximately 500m upstream
+from the confluence with the Bulkley River. Crossing 197653 is the
+responsibility of the Ministry of Transportation and Infrastructure.
+
+<br>
+
+## Background {.unnumbered}
+
+At crossing `r as.character(my_site)`, `r fpr::fpr_my_pscis_info()` is a
+`r fpr::fpr_my_bcfishpass() %>% english::ordinal()` order stream with a
+watershed area upstream of the crossing of approximately
+`r fpr::fpr_my_wshd()`km^2^. The elevation of the watershed ranges from
+a maximum of `r fpr::fpr_my_wshd(col = 'elev_max')`m to
+`r fpr::fpr_my_wshd(col = 'elev_site')`m at the crossing (Table
+\@ref(tab:tab-wshd-197653)). Upstream of crossing
+`r as.character(my_site)`, `r fpr::fpr_my_fish_sp()` have previously
+been recorded upstream [@moe2020KnownBC; @norris2020bcfishobs].
+
+<br>
+
+```{r tab-wshd-197653, eval = T}
+fpr::fpr_table_wshd_sum(site_id = my_site) %>% 
+  fpr::fpr_kable(caption_text = paste0('Summary of derived upstream watershed statistics for PSCIS crossing ', my_site, '.'),
+           footnote_text = 'Elev P60 = Elevation at which 60% of the upstream watershed area is above',
+           scroll = F)
+```
+
+<br>
+
+PSCIS stream crossing `r as.character(my_site)` was ranked as a high
+priority for follow up by Canadian Wildlife Federation because of
+significant amounts of habitat modelled upstream of the crossing. A
+total of `r fpr::fpr_my_bcfishpass(col_pull = total_lakereservoir_ha)`ha
+of lake and `r fpr::fpr_my_bcfishpass(col_pull = total_wetland_ha)`ha of
+wetland is modelled upstream and a summary of habitat modelling outputs
+is presented in Table \@ref(tab:tab-culvert-bcfp-197653). A map of the
+watershed is provided in map attachment
+[`r fpr::fpr_my_bcfishpass(col_pull = dbm_mof_50k_grid)`](%60r%20fpr::fpr_my_mapsheet()%60).
+
+`r if(identical(gitbook_on, FALSE)){knitr::asis_output("\\pagebreak")}`
+
+<br>
+
+```{r tab-culvert-bcfp-197653, eval = T}
+fpr::fpr_table_bcfp(scroll = gitbook_on) 
+```
+
+<br>
+
+## Stream Characteristics at Crossing {.unnumbered}
+
+At the time of the survey, PSCIS crossing `r as.character(my_site)` was
+non-backwatered and ranked as a
+`r fpr::fpr_my_pscis_info(col_pull = barrier_result)` to upstream fish
+passage according to the provincial protocol [@moe2011Fieldassessment]
+(Table \@ref(tab:tab-culvert-197653)). Water temperature was
+`r fpr::fpr_my_habitat_info(col_pull = 'temperature_c')`$^\circ$C, pH
+was `r fpr::fpr_my_habitat_info(col_pull = 'p_h')` and conductivity was
+`r fpr::fpr_my_habitat_info(col_pull = 'conductivity_m_s_cm')`uS/cm.
+
+<br>
+
+The outlet has large chunks of bedrock used for rip rap. There was no
```

Comment:
downstream of the outlet there were large chunks of concrete used to armour the channel.

bedrock is a type of natural rock.